tslint.json

```diff
@@ -5,6 +5,7 @@
     "interface-name": [true, "never-prefix"],
     "jsx-boolean-value": [true, "never"],
     "jsx-no-lambda": false,
+    "jsx-no-bind": false,
```

Comment:
Not sure why rebasing from master started err'ing out on this; there are areas in our code that do use `bind` and would like to avoid refactoring those right now. (This should probably be a warning anyway since its something of a premature optimization, but would still fail in CI if so.)